ambari-server/src/main/java/org/apache/ambari/server/controller/internal/BlueprintConfigurationProcessor.java

```diff
@@ -2955,41 +2954,6 @@ private void addExcludedConfigProperties(Configuration configuration, Set<String
     }
   }
 
-  /**
```

Comment:
How will existing Blueprints that set these properties be handled? 

We should make sure that if these properties are set in the older location ("cluster-env") then the Blueprint processor should move these over to "cluster-settings".

Reply:
The logic is the following:
- all properties in cluster-env which are recognized as cluster settings are copied from cluster-env to cluster settings.
- recognition is based on the default cluster settings. It is stored in cluster-settings.xml and exposed by a read only API. If a default setting is found for a cluster-env property, the property will be treated as cluster setting.
- Legacy code may still use cluster-env to get some of these properties, so as a temporary workaround the properties will remain in cluster-env too (in addition to being copied to cluster settings). Exceptions are the cases where migration has happened such case is the retry related settings.
- Users are safe the continue to use cluster-env, the properties will be automatically copied to cluster settings (provided they are recognized cluster setting properties - based on the logic above)